draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1820,44 +1820,41 @@ IDNA2003 {{RFC3490}}.
-user even if the user never visits the server directly. For example, if a user
-visits a site that contains content from a third party and then later visits
-another site that contains content from the same third party, the third party
-can track the user between the two sites.
-
-Given this risk to user privacy, some user agents restrict how third-party
-cookies behave, and those restrictions vary widly. For instance, user agents
-might block third-party cookies entirely by refusing to send Cookie header fields or
-process Set-Cookie header fields during third-party requests. They might take a less
-draconian approach by partitioning cookies based on the first-party context,
-sending one set of cookies to a given third party in one first-party context,
-and another to the same third party in another. Or they might even allow some
-third-party cookies but block others depending on user-agent cookie policy or
-user controls.
-
-This document grants user agents wide latitude to experiment with third-party
-cookie policies that balance the privacy and compatibility needs of their users.
-However, this document does not endorse any particular third-party cookie
-policy.
-
-Third-party cookie blocking policies are often ineffective at achieving their
-privacy goals if servers attempt to work around their restrictions to track
-users. In particular, two collaborating servers can often track users without
-using cookies at all by injecting identifying information into dynamic URLs.
-
-## Cookie policy {#cookie-policy}
+A cookie that is sent to a site other than the one it originated from is known as a "third-party"
+cookie. When rendering a document, a user agent can request resources from sites other than that
+where that document was obtained from (such as advertising networks). Those third-party resources
+can use cookies to track the user even if they never visit that server directly. For example, if a
+user visits a site that contains content from a third party and then later visits another site that
+contains content from the same third party, the third party can track the user between the two
+sites.
+
+Because of their inherent privacy issues, most user agents now limit third-party cookies in a
+variety of ways. Some completely block third-party cookies by refusing to process third-party
+Set-Cookie header fields and refusing to send third-party Cookie header fields. Some partition
+cookies based upon the first-party context, so that different cookies are sent depending on the
+site being browsed. Some block cookies based upon user agent cookie policy and/or user controls.
+
+It is RECOMMENDED that user agents implement third-party cookie policies that balance the
+privacy and compatibility needs of their users. However, this document does not yet endorse any
+particular third-party cookie policy.
+
+As a result, resources SHOULD NOT rely upon third-party cookies functioning reliably.
```

Comment:
Isn't this purely factual?  That is, s/SHOULD NOT/cannot/ ?

A normative statement implies that this is the way we want it to be.  I don't think that would be right.

Reply:
People assign somewhat magical qualities to 2119 language; I'm invoking that here.

Reply:
I'm saying that I don't think that invoking the mystical 2119 invocation is right.  You are saying that they are unreliable, which is a fact.  But by invoking 2119 you are saying that people shouldn't use third-party cookies, which I don't agree with.  Though perhaps the fine folks at Apple would disagree, I would argue that partitioned cookies are perfectly fine (partitioned storage more generally).  Useful even.

That's an ongoing debate that you would have this draft take a position on.

Reply:
Fair enough.